src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,57 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+)
+
+// Example: oci-factory upload -y --release tracks=1.0-22.04,risks=edge,eol=2025-06-01
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The integer-typed revision number of the image release"`
+}
+
+var opts struct {
+	// Leave `release` here for now. Should be moved into a `cmd_release.go` in phase 2.
```

Comment:
I would add a `TODO` here so that you can easily grep for it afterwards.

Reply:
Fixed.